### PR TITLE
[Fix] Upgraded run dependencies

### PR DESCRIPTION
**Pull request by viniarck**
Upgraded and pinned run dependencies to be compatible with other NApps, and also to avoid unexpected pip install resolution in the docker build.  

#### Changed

- Pinned run.in dependencies
- Pinned ``requests==2.27.0`` to be compatible with other NApps docker build
- Regenerated ``requirements/*.txt`` dependencies
- Updated version to `[2022.1b1] - "beta" - 2022-01-07`

#### Tests

In addition to the unit tests, I've also tested this in the docker build, it has built successfully, I've also used both `mef_eline` and `maintenance` branches that are under review that updated dependencies.

```
diff --git a/Dockerfile b/Dockerfile
index 476844c..8e45899 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -16,7 +16,7 @@ RUN python3 -m pip install setuptools==60.2.0
 RUN python3 -m pip install pip==21.3.1
 RUN python3 -m pip install wheel==0.37.1
 RUN python3 -m pip install https://github.com/kytos-ng/python-openflow/archive/master.zip
-RUN python3 -m pip install https://github.com/kytos-ng/kytos-utils/archive/master.zip
+RUN python3 -m pip install https://github.com/kytos-ng/kytos-utils/archive/fix/upgraded_run_deps.zip
 RUN python3 -m pip install https://github.com/kytos-ng/kytos/archive/master.zip

 RUN python3 -m pip install -e git+https://github.com/kytos-ng/storehouse@fix/install_req_deps#egg=kytos-storehouse
@@ -25,8 +25,8 @@ RUN python3 -m pip install -e git+https://github.com/kytos-ng/flow_manager#egg=k
 RUN python3 -m pip install -e git+https://github.com/kytos-ng/topology#egg=kytos-topology
 RUN python3 -m pip install -e git+https://github.com/kytos-ng/of_lldp#egg=kytos-of_lldp
 RUN python3 -m pip install -e git+https://github.com/kytos-ng/pathfinder@fix/install_req_deps#egg=kytos-pathfinder
-RUN python3 -m pip install -e git+https://github.com/kytos-ng/mef_eline@fix/install_req_deps#egg=kytos-mef_eline
-RUN python3 -m pip install -e git+https://github.com/kytos-ng/maintenance#egg=kytos-maintenance
+RUN python3 -m pip install -e git+https://github.com/kytos-ng/mef_eline@fix/upgrade_run_deps#egg=kytos-mef_eline
+RUN python3 -m pip install -e git+https://github.com/kytos-ng/maintenance@fix/install_req_deps#egg=kytos-maintenance
 RUN python3 -m pip install -e git+https://github.com/amlight/coloring@fix/install_req_deps#egg=amlight-coloring
 RUN python3 -m pip install -e git+https://github.com/amlight/sdntrace@fix/install_req_deps#egg=amlight-sdntrace
```

```
docker build -t kytos/dev -f Dockerfile . --no-cache
....
Step 31/31 : ENTRYPOINT ["/docker-entrypoint.sh"]
 ---> Running in 569c96697d65
Removing intermediate container 569c96697d65
 ---> aa91afb74e99
Successfully built aa91afb74e99
Successfully tagged kytos/dev:latest
```

### CODE REVIEW

File: kytos/utils/metadata.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 The present metadata is intended to be used mainly on the setup.
 """
-__version__ = '2021.1'
+__version__ = '2022.1b1'
```
Comment (viniarck) [inline kytos/utils/metadata.py:5]: I'm not sure if the team has agreed on which semantic versioning we'll continue with, but I ended up using using the same one in place cc'ing @italovalcy @ajoaoff @rmotitsuki let me know if you have other suggestions or if this has already been specified somewhere else.

Comment (italovalcy) [reply kytos/utils/metadata.py:5]: Yeah, using the same approach sounds good to me!

**Review by italovalcy — APPROVED** [review]
Looks good to me!

File: kytos/utils/metadata.py (hunk 1 above)
Comment (viniarck) [reply kytos/utils/metadata.py:5]: @italovalcy thanks for reviewing

File: CHANGELOG.rst [hunk 2]
```diff
@@ -23,6 +23,15 @@ Fixed
 Security
 ========
 
+[2022.1rc1] - "jovelina" - 2022-01-07
```
Comment (viniarck) [inline CHANGELOG.rst:26]: @italovalcy @ajoaoff I've set `jovelina` as the release name as we've chatted, and also upated the version as `2022.1rc1`.

**Pull request merged**